tests/cypress/plugins/index.js

```diff
@@ -1,10 +1,31 @@
 'use strict';
 
-module.exports = (on, config) => {
+const mysql = require( 'mysql' );
```

Comment:
If we're going to use this, I'd prefer that we find a way to link it to the database in the WP ENV docker container.